config

```diff
@@ -4,11 +4,15 @@ port 8080;
 
 path /echo EchoHandler {}
 path /test EchoHandler {}
+
+path /static/same StaticHandler {
+	root static;
+}
 path /static StaticHandler {
-	root static/;
+	root static;
 }
 path /special StaticHandler {
-	root static/more/;
+	root static/more;
```

Comment:
I was messing around with it when debugging and happened to change it. It doesn't seem to have any effect, but do you have a preference?